build/Build.md

```diff
@@ -0,0 +1,190 @@
+# 1. Building a container image
+
+This chapter depicts important information about building your first WSO2 container image.
+
+### Contents
+
+The following will be discussed in detail in the document.
+
+* [1.1. Why should you choose Docker as your container runtime ?](#Why-docker)
+* [1.2. Using WSO2 images](#using-wso2-images)
+* [1.3. Building from source](#building-from-source)
+* [1.4. Tips to harden your image](#tips-to-harden-your-image)
+* [1.5. Tips on Tagging](#tips-on-tagging)
+* [1.6. Vulnerability scanning and container security check](#scanning)
+* [1.7. Image signing and Verification](#signing)
+
+<br/>
+
+## 1.1. Why should you choose Docker as your container runtime ?
+
+Started just 7 years ago in 2013, Docker introduced what would become the industry standard for containers and is undeniably yet the defacto standard to build, share and run containerized apps. 
+
+<img src="../imgs/build/docker-usage.jpg" width="550">
+
+Among many factors which led WSO2 to choose and recommend Docker as its first choice for a container runtime, following is key.
```

Comment:
@DilanUA shouldn't this be `...following are key`? @chamindi-a please correct me if I am wrong.

Reply:
Fixed.